main.tf

```
@@ -48,14 +52,14 @@ resource "aws_s3_bucket" "pardot_form_handler_lambda" {
 data "archive_file" "lambda_source" {
   type = "zip"
 
-  source_dir  = "${path.module}/lambda-source"
-  output_path = "${path.module}/lambda-source/lambda-source.zip"
+  source_dir  = "${path.module}/dist"
+  output_path = "${path.module}/dist/${local.zip_name}"
 }
 
 resource "aws_s3_bucket_object" "pardot_lambda_source_object" {
   bucket = aws_s3_bucket.pardot_form_handler_lambda.id
 
-  key    = "lambda-source.zip"
+  key    = "${local.zip_name}"
```

> Warning: Interpolation-only expressions are deprecated on .terraform/modules/pardot-form-handler/main.tf line 62, in resource "aws_s3_bucket_object" "pardot_lambda_source_object":
  62:   key    = "${local.zip_name}"

```suggestion
  key    = local.zip_name
```

---

Ahhhh dang, I shoulda caught that.  I don't have TF on my personal computer so I missed that.